src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUnaryOperatorExpression.cs

```diff
@@ -2958,5 +2958,165 @@ unsafe void Method()
 
             VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
         }
+
+        [Fact]
+        public void VerifyLiftedUnaryOperators1()
+        {
+            var source = @"
+ class C
+ {
+     void F(int? x)
+     {
+         var y = /*<bind>*/-x/*</bind>*/;
+     }
+ }";
+
+            string expectedOperationTree =
+@"IUnaryOperatorExpression (UnaryOperationKind.IntegerMinus-IsLifted) (OperationKind.UnaryOperatorExpression, Type: System.Int32?) (Syntax: '-x')
+  IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Int32?) (Syntax: 'x')";
+
+            VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact]
+        public void VerifyNonLiftedUnaryOperators1()
+        {
+            var source = @"
+class C
+{
+    void F(int x)
+    {
+        var y = /*<bind>*/-x/*</bind>*/;
+    }
+}";
+
+            string expectedOperationTree =
+@"IUnaryOperatorExpression (UnaryOperationKind.IntegerMinus) (OperationKind.UnaryOperatorExpression, Type: System.Int32) (Syntax: '-x')
+  IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'x')";
+
+            VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact]
+        public void VerifyLiftedUserDefinedUnaryOperators1()
+        {
+            var source = @"
+struct C
+{
+    public static C operator -(C c) { }
+    void F(C? x)
+    {
+        var y = /*<bind>*/-x/*</bind>*/;
+    }
+}";
+
+            string expectedOperationTree =
+@"IUnaryOperatorExpression (UnaryOperationKind.OperatorMethodMinus-IsLifted) (OperatorMethod: C C.op_UnaryNegation(C c)) (OperationKind.UnaryOperatorExpression, Type: C?) (Syntax: '-x')
+  IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: C?) (Syntax: 'x')";
+
+            VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact]
+        public void VerifyNonLiftedUserDefinedUnaryOperators1()
+        {
+            var source = @"
+struct C
+{
+    public static C operator -(C c) { }
+    void F(C x)
+    {
+        var y = /*<bind>*/-x/*</bind>*/;
+    }
+}";
+
+            string expectedOperationTree =
+@"IUnaryOperatorExpression (UnaryOperationKind.OperatorMethodMinus) (OperatorMethod: C C.op_UnaryNegation(C c)) (OperationKind.UnaryOperatorExpression, Type: C) (Syntax: '-x')
+  IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: C) (Syntax: 'x')";
+
+            VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact]
+        public void VerifyLiftedBinaryOperators1()
```

Comment:
The `BinaryOperator` tests are in `_IUnaryOperatorExpression.cs`. Should the file be renamed? Similar comment for VB.